applications/nrf_desktop/doc/usb_state.rst

```diff
@@ -43,6 +43,13 @@ The USB stack is selected by enabling one of the following Kconfig choice option
   The HID boot protocol integration is not yet fully tested and may not work properly.
   The secondary image slot background erase in :ref:`nrf_desktop_dfu` may cause missing USB HID subscriptions after a USB cable is attached.
 
+The USB next stack is the only stack that supports the USB High-Speed.
+You can configure your preferred USB HID polling rate using the ``in-polling-rate`` property of DTS node compatible with ``zephyr,hid-device``.
+The lowest polling rate that is supported by the USB High-Speed is ``125`` us which corresponds to ``8`` kHz report rate.
```

Comment:
Section `Low latency device configuration` also  mentions `CONFIG_USB_HID_POLL_INTERVAL_MS` that is only for legacy stack.
Maybe it should be moved there, or parts of this regarding the poll interval?